tox.ini

```diff
@@ -43,7 +43,6 @@ deps =
   flake8
   isort
   black
-  psutil
```

Comment:
Is this not causing issues in travis build making perf metrics tests to fail?

Reply:
`psutil` does not seem to be needed for the `lint` environment.

https://travis-ci.org/github/lzchen/opentelemetry-exporters-python/builds/667556691

Reply:
Good, I added this because of some error but maybe it was related to my local box setup